datafusion/core/tests/memory_limit.rs

```diff
@@ -55,118 +55,110 @@ fn init() {
 
 #[tokio::test]
 async fn oom_sort() {
-    TestCase::new(
-        "select * from t order by host DESC",
-        vec![
+    TestCase::new()
+        .with_query("select * from t order by host DESC")
```

Comment:
the major change is that by using a builder pattern, now the tests are more explicit about what the values are